### PR TITLE
Added mbi 21.04.2 rn

### PR DESCRIPTION
**Pull request by sbrizini**
## Description

Please include a short resume of the changes and what is the purpose of the PR.

Any relevant information should be added to help reviewers.

## Target series

- [ ] 20.04.x
- [ ] 20.10.x
- [x] 21.04.x
- [ ] 21.10.x (master)

### CODE REVIEW

File: en/releases/centreon-commercial-extensions.md [hunk 1]
```diff
@@ -31,6 +31,21 @@ If you have feature requests or want to report a bug, please contact support.
...
+
+`July 15, 2021`
+
+#### Improvements
+
+- Optimized widget requests
+
+#### Bugfixes
+
+- Removed old credentials from the repo
```
Comment (Nohzoh) [inline en/releases/centreon-commercial-extensions.md:44]: This is just creds used for tests, it has no impact on the product itself.
Not sure we should put that in the RN.
```suggestion
```

File: fr/releases/centreon-commercial-extensions.md [hunk 2]
```diff
@@ -31,6 +31,20 @@ commerciales, veuillez contacter le support.
 
 ## Centreon MBI
 
+### 21.04.2
+
+`15 juillet 2021`
+
+#### Améliorations
+
+- Requetes optimisées pour les widgets
```
Comment (thiuyendang) [inline fr/releases/centreon-commercial-extensions.md:40]:
```suggestion
- Requêtes optimisées pour les widgets
```

File: fr/releases/centreon-commercial-extensions.md [hunk 3]
```diff
@@ -31,6 +31,20 @@ commerciales, veuillez contacter le support.
...
+
+#### Correctifs
+
+- Correctif additionnel de l'éxecution immédiate des tâches
```
Comment (thiuyendang) [inline fr/releases/centreon-commercial-extensions.md:44]:
```suggestion
- Correctif additionnel de l'exécution immédiate des tâches
```

**Review by thiuyendang — CHANGES_REQUESTED** [review]

**Review by Nohzoh — APPROVED** [review]

File: fr/releases/centreon-commercial-extensions.md [hunk 4]
```diff
@@ -31,6 +31,20 @@ commerciales, veuillez contacter le support.
...
+`15 juillet 2021`
+
+#### Améliorations
+
+- Requêtes optimisées pour les widgets
+
+#### Correctifs
+
+- Correctif additionnel de l'exécution immédiate des tâches
+- Correction des noms de BA manquant dans le rapport BV-BA-Availabilities-1
```
Comment (thiuyendang) [inline fr/releases/centreon-commercial-extensions.md:45]:
```suggestion
- Correction des noms de BA manquants dans le rapport BV-BA-Availabilities-1
```

**Review by thiuyendang — CHANGES_REQUESTED** [review]

**Review by thiuyendang — APPROVED** [review]

**Pull request merged**